src/containers/InboxPage/InboxPage.js

```diff
@@ -167,6 +167,11 @@ export const InboxItem = props => {
           ) : hasPricingData && showStock ? (
             <FormattedMessage id="InboxPage.quantity" values={{ quantity }} />
           ) : null}
+          {availabilityType == 'multipleSeats' ? (
```

Comment:
```suggestion
          {availabilityType === 'multipleSeats' ? (
```

But now that this branch is on top of 'seats' branch, you could do something like this instead

`availabilityType === AVAILABILITY_MULTIPLE_SEATS`

https://github.com/sharetribe/web-template/pull/502/files#diff-b10252d2949b95c49dad71e029508ee32996c4ffdff79774484f4ed0d0c958a2R342